webroot/js/utils/chat.js

```diff
@@ -278,3 +278,15 @@ export function convertOnPaste( event = { preventDefault() {} }) {
     document.execCommand('insertText', false, value);
   }
 }
+
+export function formatTimestamp(sentAt) {
+  sentAt = new Date(sentAt);
```

Comment:
In case `sentAt` is falsey, let's return an empty string '' to start off with;

Reply:
Done